app/scripts/actions/index.js

```diff
@@ -253,6 +253,29 @@ export const REFRESH_TOKEN_INFLIGHT = 'REFRESH_TOKEN_INFLIGHT';
 
 export const SET_TOKEN = 'SET_TOKEN';
 
+export const API_VERSION = 'API_VERSION';
+export const API_VERSION_INFLIGHT = 'API_VERSION_INFLIGHT';
+export const API_VERSION_ERROR = 'API_VERSION_ERROR';
+
+export const SET_API_VERSION_COMPAT_ERROR = 'SET_API_VERSION_COMPAT_ERROR';
+
+export const getApiVersion = () => wrapRequest(
+  null, get, 'version', API_VERSION);
+
+export const validateApiVersion = (apiVersion, compatibleList, dispatch) => {
+  const isCompatible = compatibleList.indexOf(apiVersion) > -1;
+  if (!isCompatible) {
+    dispatch({
+      type: SET_API_VERSION_COMPAT_ERROR,
+      payload: {
+        isCompatible,
+        apiVersion,
+        compatibleList
```

Comment:
It doesn't seem like we need the compatibility list in this payload?

Reply:
Good point - I wanted to include that information in the error/warning and I wasn't sure if I wanted to write that error message in this function or in the reducer.